README.md

```diff
@@ -1 +1,33 @@
-# hw01
\ No newline at end of file
+# Homework 1
+
+### About Me
+
+My name is **Jonathan Johnson** or *JJ* for short. 
+
+![](https://i.imgur.com/5JGh0bLm.png)
```

Comment:
This is a great alternative way to embed an image in Markdown. In order to embed in image locally, you can: 
1. Upload the image to your homework repository, or
2. Add the image to the same file as your README.md and push it to GitHub.